### ISSUE TITLE
Magento 2.1.11 minimum quantity validation message not showing

### ISSUE THREAD

**Issue by jayanta88** [magento/magento2#13582]
Hi,
Magento default validation message for "validate-item-quantity" not showing. It also not found in /lib/web/mage/validation.js
![image](https://user-images.githubusercontent.com/16479533/36025866-a6e3aece-0dba-11e8-9904-01d3f3329941.png)

Is there any reason? Or can be fixed in next version?

Thanks

**Comment by jayanta88** [comment]
How can I show a different message for "minAllowed", "maxAllowed" etc. ?

**Comment by magento-engcom-team** [comment]
@jayanta88, thank you for your report.
We've acknowledged the issue and added to our backlog.

**Comment by dyron** [comment]
You can use this mixin waiting for the fix:
```javascript
define([
    'jquery',
    'mage/validation'
], function ($) {
    'use strict';

    return function () {
        $.validator.addMethod(
            'validate-item-quantity',
            function (value, element, params) {
                // obtain values for validation
                var minAllowed,
                    maxAllowed,
                    qtyIncrements,
                    validator = this,
                    qty = $.mage.parseNumber(value),
                    isMinAllowedValid = typeof params.minAllowed === 'undefined' ||
                        qty >= (minAllowed = $.mage.parseNumber(params.minAllowed)),
                    isMaxAllowedValid = typeof params.maxAllowed === 'undefined' ||
                        qty <= (maxAllowed = $.mage.parseNumber(params.maxAllowed)),
                    isQtyIncrementsValid = typeof params.qtyIncrements === 'undefined' ||
                        qty % (qtyIncrements = $.mage.parseNumber(params.qtyIncrements)) === 0;

                validator.validateMessage = $.mage.__('We don\'t have as many "%1" as you requested.').replace('%1', qty);

                if (!isMinAllowedValid) {
                    validator.validateMessage =
                        $.mage.__('The fewest you may purchase is %1.').replace('%1', minAllowed);
                }
                if (!isMaxAllowedValid) {
                    validator.validateMessage =
                        $.mage.__('The most you may purchase is %1.').replace('%1', maxAllowed);
                }
                if (!isQtyIncrementsValid) {
                    validator.validateMessage =
                        $.mage.__('You can buy this product only in quantities of %1 at a time.').replace('%1', qtyIncrements);
                }
                return isMaxAllowedValid && isMinAllowedValid && isQtyIncrementsValid && qty > 0;
            },
            function () {
                return this.validateMessage;
            }
        );
    };
});
```

**Comment by jayanta88** [comment]
Hi Dyron - 
Thanks for your solution. I had fixed it with custom validation by extending the mage/validation.
I couldn't able to pass different message which you achieved with validator.validateMessage. It helps me.

Thanks

**Comment by bordeo** [comment]
I made a pull request to resolve this issue 20 days ago. 
Someone could make a check and merge it?
Thank you

**Comment by magento-engcom-team** [comment]
Hi @jayanta88. Thank you for your report. 
The issue has been fixed in magento/magento2#13942 by @bordeo in 2.2-develop branch
Related commit(s):
 - [63c2de8b19d341158b000b87d9ae7cde9d6b9c1c](https://github.com/magento/magento2/commit/63c2de8b19d341158b000b87d9ae7cde9d6b9c1c)
 - [865773b810090205d265e0c1a88493aeca382741](https://github.com/magento/magento2/commit/865773b810090205d265e0c1a88493aeca382741)
 - [3fdaa11a4375fe2c434ff665cd46f51af980975b](https://github.com/magento/magento2/commit/3fdaa11a4375fe2c434ff665cd46f51af980975b)
 - [22539f972e32128aeeec0f65aebb1f29879f8231](https://github.com/magento/magento2/commit/22539f972e32128aeeec0f65aebb1f29879f8231)

The fix will be available with the upcoming 2.2.5 release.

**Comment by sidolov** [comment]
Hi @jayanta88. Thank you for your report. 
The issue has been fixed in magento/magento2#14508 by @mastiuhin-olexandr in 2.3-develop branch
Related commit(s):
 - [a2c1a64ef86d99162511beb8da40bf68ca9bcd5a](https://github.com/magento/magento2/commit/a2c1a64ef86d99162511beb8da40bf68ca9bcd5a)
 - [2596ad5d3d427cb200e79fbd3eeaff156c7ba4b6](https://github.com/magento/magento2/commit/2596ad5d3d427cb200e79fbd3eeaff156c7ba4b6)
 - [2be7da9d24744452bb7192b154d084f6f90ba619](https://github.com/magento/magento2/commit/2be7da9d24744452bb7192b154d084f6f90ba619)
 - [eae1c0323ca370b4907198c47a6ce07d35324307](https://github.com/magento/magento2/commit/eae1c0323ca370b4907198c47a6ce07d35324307)

The fix will be available with the upcoming 2.3.0 release.
